lib/clientv3/node_e2e_test.go

```diff
@@ -221,6 +221,47 @@ var _ = testutils.E2eDatastoreDescribe("Node tests", testutils.DatastoreEtcdV3,
 			bconfig, err := c.BGPConfigurations().Get(ctx, nodeConfigName, options.GetOptions{})
 			Expect(bconfig).Should(BeNil())
 		})
+
+	})
+
+	var _ = testutils.E2eDatastoreDescribe("Node tests (kdd)", testutils.DatastoreK8s, func(config apiconfig.CalicoAPIConfig) {
```

Comment:
Indentation looks wrong here.  Is it because this is positioned _inside_ the `E2eDatastoreDescribe` for "Node tests (etcdv3)"?  Surely that's wrong, if so?

Reply:
Yes... good spot!

I suppose I trust `goimports` a bit too much :)